### PR TITLE
Add OpenVINO notebook to README

### PR DESCRIPTION
**Pull request by helena-intel**
Add OpenVINO notebook to notebooks README

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -22,3 +22,4 @@ You can find here a list of the notebooks, leveraging the usage of the Optimum I
...
+| [How to quantize a question-answering model with Post Training Quantization in NNCF and do inference with OpenVINO](https://github.com/huggingface/optimum-intel/blob/main/notebooks/openvino/question_answering_quantization.ipynb) | Show how to apply post-training quantization on a question-answering BERT model using [NNCF](https://github.com/openvinotoolkit/nncf) and do accelerated inference with OpenVINO through optimum-intel | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/optimum-intel/blob/main/notebooks/openvino/question_answering_quantization.ipynb)| [![Open in AWS Studio](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/huggingface/optimum-intel/blob/main/notebooks/openvino/question_answering_quantization.ipynb)|
```
Comment (AlexKoff88) [inline notebooks/README.md:25]: I am just wondering if these notebooks are discoverable from the Optimum documentation.

**Review by AlexKoff88 — APPROVED** [review]

File: notebooks/README.md (hunk 1 above)
Comment (echarlaix) [reply notebooks/README.md:25]: I can add it in optimum's documentation !

Comment (echarlaix) [inline notebooks/README.md:25]:
```suggestion
| [How to quantize a question answering model with OpenVINO NNCF](https://github.com/huggingface/optimum-intel/blob/main/notebooks/openvino/question_answering_quantization.ipynb) | Show how to apply post-training quantization on a question answering model using [NNCF](https://github.com/openvinotoolkit/nncf) and to accelerate inference with OpenVINO| [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/optimum-intel/blob/main/notebooks/openvino/question_answering_quantization.ipynb)| [![Open in AWS Studio](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/huggingface/optimum-intel/blob/main/notebooks/openvino/question_answering_quantization.ipynb)|
```

**Review by echarlaix — APPROVED** [review]
Thanks for adding it !

**Pull request merged**